go.mod

```diff
@@ -0,0 +1,13 @@
+module go4.org
+
+go 1.13
```

Comment:
Does anything in here actually require Go 1.13?

Reply:
The code doesn't require Go 1.13.

The minimum Go version for go module is Go 1.11 but with Go 1.11 or Go 1.12  the `go.mod` file doesn't have the same content:

```go
require (
	cloud.google.com/go v0.53.0
	cloud.google.com/go/storage v1.5.0
	github.com/rwcarlsen/goexif v0.0.0-20190401172101-9e8deecbddbd
	golang.org/x/net v0.0.0-20200222125558-5a598a2470a0
	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e // indirect
	golang.org/x/sys v0.0.0-20200223170610-d5e6a3e2c0ae
	google.golang.org/api v0.17.0
)
```
The `golang.org/x/sync` line is added with Go 1.11 or Go 1.12 but dropped with Go 1.13.

This variation can cause validation of Go Modules files to fail.
I updated the Travis CI file to do that (Travis CI seems not running on PR)

I can put the version that you want.